packages/flutter/test/widgets/focus_scope_test.dart

```diff
@@ -1079,6 +1079,73 @@ void main() {
       expect(focusNodeB.hasPrimaryFocus, isFalse);
       expect(focusNodeA.hasPrimaryFocus, isTrue);
     });
+
+    testWidgets("FocusScope doesn't update the focusNode attributes when the widget updates if withExternalFocusNode is used", (WidgetTester tester) async {
+      final GlobalKey key1 = GlobalKey(debugLabel: '1');
+      final FocusScopeNode focusScopeNode = FocusScopeNode();
+      bool? keyEventHandled;
+      // ignore: prefer_function_declarations_over_variables
+      final FocusOnKeyCallback handleCallback = (FocusNode node, RawKeyEvent event) {
```

Comment:
is it possible to use inline functions to avoid triggering the linter?

Reply:
No, because I want to verify below that the value of the callback attribute hasn't changed, and even if I used two identical inlines, they would be different values.

Reply:
By inline I meant declaring functions inside of the test method (not lambdas):
```dart
KeyEventResult handleCallback(FocusNode node, RawKeyEvent event) { 
}
```
Or is that not going to work because tear-offs are under the hood wrapped in anonymous functions so `focusNode.onKey != handleCallback`?

Reply:
Ahh, I see.  Yeah, I'll try that.

Reply:
Done.